hudi-timeline-service/src/main/java/org/apache/hudi/timeline/service/handlers/MarkerHandler.java

```diff
@@ -274,7 +269,7 @@ private CompletableFuture<String> finishCreateMarkerFuture(Context context, Stri
     MarkerCreationFuture future = new MarkerCreationFuture(context, markerDir, markerName);
     try {
       future.complete(jsonifyResult(
-          future.getContext(), future.isSuccessful(), metricsRegistry, new ObjectMapper(), LOG));
```

Comment:
This was creating a new `ObjectMapper` per invocation which is very bad for performance. The object is meant to be reused across threads